.travis.yml

```diff
@@ -78,7 +82,7 @@ script:
     - ./configure --cache-file=../config.cache $BITCOIN_CONFIG_ALL $BITCOIN_CONFIG || ( cat config.log && false)
     - make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && make $GOAL V=1 ; false )
     - export LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/depends/$HOST/lib
-    - if [ "$RUN_TESTS" = "true" ]; then travis_wait 50 make $MAKEJOBS check VERBOSE=1; fi
+    - if [ "$RUN_TESTS" = "true" ]; then travis_wait 15 make $MAKEJOBS check VERBOSE=1; fi
```

Comment:
What is the reasoning behind the change from 50 to 15 here?

Reply:
It was raised from 30 to 50. I think it should be dropped to 30 again.

Reply:
30 makes sense

Reply:
30 sounds reasonable.